ui/src/app.js

```diff
@@ -272,16 +272,24 @@ function processData (data) {
 }
 
 function translate (data, fetchArgs) {
+  if (data === "")
+    return data
+
   try {
     return fetchArgs
       ? locale.translate(data).fetch(...fetchArgs)
       : locale.translate(data).fetch()
   } catch (error) {
     if (!defaultLocale) console.error('Error while translating: ', error)
     else {
-      return fetchArgs
-        ? defaultLocale.translate(data).fetch(...fetchArgs)
-        : defaultLocale.translate(data).fetch()
+      try {
+        return fetchArgs
+          ? defaultLocale.translate(data).fetch(...fetchArgs)
+          : defaultLocale.translate(data).fetch()
+      } catch (e) {
+        console.error('Error while translating: ', e)
+        return data
+      }
```

Comment:
Also any reason for the default translation to fail? Won't this hide away a issue on the translation itself?

Reply:
There seams that exist a few elements that fail default translation (those element have `baseTranslation` assigned to them, I forgot to fix that). But, same reason as above, I wanted to make `translate` function safer.